CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Default cloud channel to '7'
```

Comment:
Add categories here I guess this is a Changed category and the other one is a Added

Reply:
Right 👍

Reply:
Done!